coinmarketcap/test/adapter.test.ts

```diff
@@ -1,9 +1,13 @@
-const { assert } = require('chai')
-const { assertSuccess, assertError } = require('@chainlink/adapter-test-helpers')
-const { execute } = require('../adapter')
+import { assert } from 'chai'
+import { Requester } from '@chainlink/external-adapter'
+import { assertSuccess, assertError } from '@chainlink/adapter-test-helpers'
+import { AdapterRequest } from '@chainlink/types'
+import { makeExecute } from '../src/adapter'
 
 describe('execute', () => {
   const jobID = '1'
+  const execute = makeExecute()
+  process.env.API_KEY = 'test_api_key'
```

Comment:
Shouldn't we use an actual API_KEY here?

Reply:
We can't commit API keys to the public repository.

But, you are right that integration tests set like this are not too usable. We want to somehow start integration tests for all adapters at once, but then how do we inject a different API key to each.

One possible solution would be to have a `secrets.json` configured with all secrets required for integration tests. Then every test can read from there.